http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/MultipartInputstreamIT.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import static io.quarkus.ts.http.advanced.reactive.MultipartResource.MULTIPART_FORM_PATH;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+import jakarta.ws.rs.core.MediaType;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class MultipartInputstreamIT {
+
+    @QuarkusApplication(classes = { MultipartResource.class, MediaTypeResource.class }, properties = "oidcdisable.properties")
+    static RestService app = new RestService();
+
+    @Tag("https://github.com/quarkusio/quarkus/issues/43364")
+    @Test
+    public void testInputStreamWithoutMediaType() {
+        String testContent = "test content without media type";
+        InputStream fileStream = new ByteArrayInputStream(testContent.getBytes(StandardCharsets.UTF_8));
+        String description = "Test Description";
+
+        app.given()
+                .multiPart("file", "test.txt", fileStream)
+                .multiPart("description", description)
+                .when()
+                .post(MULTIPART_FORM_PATH + "/inputstream/without-media-type")
+                .then()
+                .statusCode(200)
+                .body(equalTo("Received description: Test Description with file content: " + testContent));
```

Comment:
Not needed probably just my preference, but wouldn't be better to substitute `Test Description` with `description` variable as to have it defined in one place.